pkgs/os-specific/linux/wpa_supplicant/default.nix

```diff
@@ -19,6 +19,11 @@ stdenv.mkDerivation rec {
       url = "https://w1.fi/security/2019-7/0001-AP-Silently-ignore-management-frame-from-unexpected-.patch";
       sha256 = "15xjyy7crb557wxpx898b5lnyblxghlij0xby5lmj9hpwwss34dz";
     })
+    (fetchpatch {
+      name = "";
```

Comment:
Can you set a name or add a comment why we want this patch?

Reply:
Oops, yeah.